pkg/apis/build/v1alpha1/build_types.go

```diff
@@ -71,6 +71,10 @@ type BuildSpec struct {
 	// More info: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/
 	// +optional
 	NodeSelector map[string]string `json:"nodeSelector,omitempty"`
+
+	// Time after which the build times out.
+	// Defaults to 10minutes.
```

Comment:
typo: `10 minutes`.

This should maybe also link to Go's `ParseDuration` documentation to clarify the expected format: https://golang.org/pkg/time/#ParseDuration

Should we enforce a _maximum_ timeout too? Today, GCB limits builds to 24h for our own implementation purposes, though I suppose there's no strong reason to carry that limitation over to Knative Build. OTOH, it's easier to remove a restriction than to add one later. WDYT?